Samples.Swift/Samples.Swift/Extensions/String+Extensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  String+Extensions.swift
+//  Samples.Swift
+//
+//  Created by Jax DesMarais-Leder on 3/25/21.
+//  Copyright © 2021 PayPal. All rights reserved.
+//
+
+import UIKit
+
+extension String{
+  func convertDoubleToCurrency(withQuantity quantity: String) -> String{
+    guard let amount = Double(self) else { return "" }
+    guard let quantity = Double(quantity) else { return "" }
+       let numberFormatter = NumberFormatter()
+       numberFormatter.numberStyle = .currency
+       numberFormatter.locale = Locale(identifier: "en_US")
+    return numberFormatter.string(from: NSNumber(value: amount * quantity)) ?? ""
```

Comment:
I think it's clearer to return an optional string here, instead of an empty string